### DISCUSSION TITLE
Batch resolve references

### DISCUSSION THREAD

**Discussion by basbebe** [keepassxreboot/keepassxc#8864] [Q&A] [answered]
I have a lot of entries that each reference another entry.
I would like to make these "_reference entries_" the new "_main entries_" and afterwards delete the former main entries (since the reference entries have additional information).

Is there an option to somehow (batch) resolve / expand references?

**Comment by droidmonkey** [comment, marked as answer]
Just delete the main entries, it will prompt you to overwrite values of the references with the actual value

**Reply by basbebe** [reply]
Lovely!
I wasn't aware of that, thanks!
